src/components/CodeInput.tsx

```diff
@@ -103,9 +103,8 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     padding: Spacing.Regular16,
     overflow: 'hidden',
-    backgroundColor: colors.gray1,
     borderWidth: 1,
-    borderColor: colors.gray2,
+    borderColor: colors.black,
```

Comment:
wouldn't it be nice if we had a shared component for this? I've been making these changes everywhere too 😄